openshift/kustomize/services/image/base/deploy.yaml

```diff
@@ -0,0 +1,189 @@
+# How the app will be deployed to the pod.
+kind: DeploymentConfig
+apiVersion: apps.openshift.io/v1
+metadata:
+  name: image-service
+  namespace: default
+  annotations:
+    description: Defines how to deploy image-service
+  labels:
+    name: image-service
+    part-of: tno
+    version: 1.0.0
+    component: image-service
+    managed-by: kustomize
+    created-by: jeremy.foster
+spec:
+  replicas: 1
+  selector:
+    name: image-service
+    part-of: tno
+    component: image-service
+  strategy:
+    rollingParams:
+      intervalSeconds: 1
+      maxSurge: 25%
+      maxUnavailable: 25%
+      timeoutSeconds: 600
+      updatePeriodSeconds: 1
+    type: Rolling
+  template:
+    metadata:
+      name: image-service
+      labels:
+        name: image-service
+        part-of: tno
+        component: image-service
+    spec:
+      volumes:
+        - name: av-storage
+          persistentVolumeClaim:
+            claimName: av-storage
+        - name: ssh-key 
+          secret:  
+            secretName: ssh-key # name of the Secret
+      containers:
+        - name: image-service
+          image: ""
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+              protocol: TCP
+          volumeMounts:
+            - name: av-storage
+              mountPath: /data  
+            - name: ssh-key  
+              mountPath: /keys
+              readOnly: true
+          resources:
+            requests:
+              cpu: 20m
+              memory: 250Mi
+            limits:
+              cpu: 100m
+              memory: 500Mi
+          env:
+            # .NET Configuration
+            - name: ASPNETCORE_ENVIRONMENT
+              value: Production
+            - name: ASPNETCORE_URLS
+              value: http://+:8080
+
+            # Common Service Configuration
+            - name: Service__ApiUrl
+              valueFrom:
+                configMapKeyRef:
+                  name: services
+                  key: API_HOST_URL
+
+            - name: Auth__Keycloak__Authority
+              valueFrom:
+                configMapKeyRef:
+                  name: services
+                  key: KEYCLOAK_AUTHORITY
+            - name: Auth__Keycloak__Audience
+              valueFrom:
+                configMapKeyRef:
+                  name: services
+                  key: KEYCLOAK_AUDIENCE
+            - name: Auth__Keycloak__Secret
+              valueFrom:
+                configMapKeyRef:
+                  name: services
+                  key: KEYCLOAK_CLIENT_SECRET
+
+            - name: Kafka__Admin__ClientId
+              valueFrom:
+                configMapKeyRef:
+                  name: image-service
+                  key: KAKFA_CLIENT_ID
+            - name: Kafka__Admin__BootstrapServers
+              valueFrom:
+                configMapKeyRef:
+                  name: services
+                  key: KAFKA_BOOTSTRAP_SERVERS
+
+            - name: Kafka__Consumer__ClientId
```

Comment:
I don't think the Image Service needs this `Kafka__Consumer__*` environment variables.